db/migrations/20201020182259_backfill_kpack_lifecycle_data_association_to_droplets.rb

```diff
@@ -0,0 +1,18 @@
+Sequel.migration do
```

Comment:
I hate making this sort of migration -- lots of potential for performance and data integrity issues -- so would appreciate a close look!

🕵️

Reply:
This migration is now gone. Resolving.